pages/error/index.html

```diff
@@ -3,11 +3,13 @@
 
 <head>
 	<link rel="stylesheet" href="../pagebase.css" />
+	<link rel="stylesheet" href="/error.css" />
+	<link rel="stylesheet" href="../../css/darkMode.css"/> <!-- This is how you add Dark Mode! Please do it -->
 	<link rel="stylesheet" href="../../ext/font-awesome-4.4.0/css/font-awesome.min.css" />
 	<script defer src="error.js"></script>
 </head>
 
-<body style="background-color: #eee">
```

Comment:
Removed Background-color from Error and Crash to put them in separate files because they were conflicting with the darkMode.css